src/main/java/com/aws/greengrass/secretmanager/LocalStoreMap.java

```diff
@@ -45,7 +45,7 @@
 
 public class LocalStoreMap {
     private final Logger logger = LogManager.getLogger(LocalStoreMap.class);
-    private final ConcurrentHashMap<String, Labels> secrets;
+    private final HashMap<String, Labels> secrets;
```

Comment:
```suggestion
    private final Map<String, Labels> secrets;
```
same in other places